### ISSUE TITLE
Can I remove detection part while predicting keypoints of person using yolo11s-pose?

### ISSUE THREAD

**Issue by hvmatrix** [ultralytics/ultralytics#17992]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hi,
I am using  yolo11s-pose for person's keypoint detection. But before that I'm using yolo11s for detection of person and while detecting bounding box, I am cropping and then save the image of that person in different folder. After that, I'm running yolo11s-pose for keypoint detection.

Here as we know yolo11s-pose first detecting person and then estimating keypoints. Is it possible to remove detection part while predicting because I am already giving cropped image of person so no need to detect it first. If yes, then please guide me for that. 

Thank you

**Comment by UltralyticsAssistant** [comment]
👋 Hello @hvmatrix, thank you for your interest in Ultralytics 🚀! We recommend visiting the [Docs](https://docs.ultralytics.com) for detailed information about YOLO models, including [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples, which might help answer your question or provide insights.

If this is a ❓ Question about customizing YOLO for your use case, please provide additional details if relevant, such as:

- Any custom modifications you've applied to the model or codebase
- Logs or screenshots of results
- Sample inputs and outputs

If this is a 🐛 Bug Report and you suspect an issue with the current implementation, we kindly request a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) that demonstrates the issue, as it will greatly assist in debugging.

While you're here, consider joining the vibrant Ultralytics community for more support and discussions. You can engage with us in several ways:
- Real-time chat: [Discord](https://discord.com/invite/ultralytics) 🎧
- In-depth discussions: [Discourse](https://community.ultralytics.com)
- Community threads: [Subreddit](https://reddit.com/r/Ultralytics)

## Upgrade

To ensure you're running the latest features and patches, upgrade to the latest `ultralytics` package with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) installed. Use a [**Python>=3.8**](https://www.python.org/) environment and make sure [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) is set up:

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run in several pre-configured environments with all dependencies (including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/)) preinstalled:

- **Notebooks** with free GPU access:
  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a>
  <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>
  <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are currently passing, verifying the correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

This is an automated response to help streamline initial issue triaging. An Ultralytics engineer will review your question and assist you soon. Thank you for your patience! 😊

**Comment by Y-T-G** [comment]
No, you can't. And you shouldn't be providing a cropped image to it as the performance will be bad because it was not trained on cropped images, but on the whole image.

**Comment by hvmatrix** [comment]
But when m giving cropped image of person actually it's working well in my case so that's the question whether shall I remove detection part or not.

**Comment by Y-T-G** [comment]
You should use YOLO-Pose for both detection and keypoint. 

You can't remove detection from it. It's part of the architecture.

**Comment by hvmatrix** [comment]
Ok sir. Thanks for clarification.

**Comment by glenn-jocher** [comment]
You're welcome! If you have any other questions, feel free to ask. The YOLO community and Ultralytics team are here to help!
